.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+name: Release
+on:
+  release:
+    types:
+      # works for building pre-releases
+      - published
+      # works for releases and changing from pre-release to release
+      - released
+
+env: 
+ 
+  DOCKER_IMAGE_NAME: newrelic/nri-kube-events
+  RELEASE_VERSION: ${{ github.event.release.name }}
```

Comment:
I have a slight preference for `${{ github.event.release.tag_name }}`, as the release name is a bit more arbitrary (and visible only in GH UI, as opposed to the tag which is built into the git tree)